tracks/python/exercises/two-fer/mentoring.md

```diff
@@ -76,3 +76,20 @@ def two_fer(name=None):
     else:
         return 'One for you, one for me.'
 ```
+
+#### Variadic Argument List
+
+We can use a variadic argument list to solve the problem
+
+```python
+def two_fer(*name):
+    if name:
```

Comment:
`if` statement is redundant. The following is functionally equivalent:

```Python
def two_fer(*name):
  for var in name:
    return f'One for {var}, one for me.'
  else:
    return f'One for you, one for me.'
```

Reply:
You are right.  I pasted in what the student wrote, and was regretting it while rereading this morning.

Reply:
@jeffdparker You marked this as resolved, but did not commit any changes or state an intent to leave as-is.

Reply:
Your change seems to be in place - When I review changes, the approve button is not enabled.

Reply:
Honestly, I have no memory of doing the right thing.